static/js/subs.js

```diff
@@ -26,6 +26,29 @@ function settings_button_for_sub(sub) {
     return $(".subscription_settings[data-stream-id='" + id + "'] .subscribe-button");
 }
 
+function get_row_data(row) {
+    var row_id = row.attr('data-stream-id');
+    if (row_id) {
+        var row_object = stream_data.get_sub_by_id(row_id);
+        return [row_id, row_object];
+    }
+}
+
+function get_active_data() {
+    var active_row = $('div.stream-row.active');
+    var valid_active_id = active_row.attr('data-stream-id');
+    var active_tab = $('.subscriptions-container').find('div.ind-tab.selected');
+    return [active_row, valid_active_id, active_tab];
```

Comment:
I think the code might be a lot more readable if you returned an object rather than a list in these two methods